cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -131,6 +133,35 @@ def test_embedded_ansible_repository_crud(ansible_repository, wait_for_ansible):
     assert view.description.value == updated_description
 
 
+@pytest.mark.rhel_testing
+@pytest.mark.tier(1)
+def test_embedded_ansible_repository_branch_crud(appliance, wait_for_ansible):
```

Comment:
Q:
As its crud test. it's possible to add update some parameters?

Reply:
Updating name's and all it is already covered in the Repository_CRUD tests.